docs-src/cloud/support-training.md

```diff
@@ -0,0 +1,17 @@
+---
+id: support-training
+title: What type of training is available?
+sidebar_label: Training
```

Comment:
I would refer to this as resources or "general developer resources" or something.
"Training" doesn't feel like the right label here.

Reply:
Updating to `What type of developer resources exist`